opentasks-provider/src/main/java/org/dmfs/provider/tasks/TaskProvider.java

```diff
@@ -1298,7 +1313,10 @@ protected void onEndTransaction(boolean callerIsSyncAdapter)
     {
         super.onEndTransaction(callerIsSyncAdapter);
         Intent providerChangedIntent = new Intent(Intent.ACTION_PROVIDER_CHANGED, TaskContract.getContentUri(mAuthority));
-        updateNotifications();
+        if (mChanged)
```

Comment:
Doesn't `mChanged` have to be reset to `false` somewhere?

Reply:
Seems reasonable to me.